### DISCUSSION TITLE
Waiting Close Channel - ChansStatusDataLoss

### DISCUSSION THREAD

**Discussion by sperolightning** [lightningnetwork/lnd#7183] [Troubleshooting]
I have the following issue: 
it happened that there was a power failure and subsequently it appeared that there was a "waiting close channel". The node is synchronized and I tried to perform the forced closure command as read in other posts:
lncli closechannel – force “channel_points” 1 
but the following message appears 
rpc error appears: code = Unknown desc = cannot close channel with state: ChanStatusLocalDataLoss

Other user resolved by contacting the other node forcing closure. In this case the node is Kraken, and I imagine it is not easy to contact and solve the issue. I wonder if there is a software solution to solve the problem? 
Thanks for your advices

**Comment by guggero** [comment]
Duplicate, see https://github.com/lightningnetwork/lnd/discussions/7187.

Also, please don't cross post. The discussion was the correct place for this.
